core/src/banking_stage/leader_slot_metrics.rs

```diff
@@ -52,6 +54,53 @@ pub(crate) struct ProcessTransactionsSummary {
 
     // Breakdown of all the transaction errors from transactions passed for execution
     pub error_counters: TransactionErrorMetrics,
+
+    pub scheduled_min_prioritization_fees: usize,
+    pub scheduled_max_prioritization_fees: usize,
+}
+
+// Metrics describing prioritization fee information for each transaction storage before processing transactions
+#[derive(Debug, Default)]
+struct LeaderPrioritizationFeesMetrics {
+    // minimum prioritization fees in the MinMaxHeap
+    min_prioritization_fees_per_cu: u64,
+    // maximum prioritization fees in the MinMaxHeap
+    max_prioritization_fees_per_cu: u64,
+}
+
+impl LeaderPrioritizationFeesMetrics {
+    fn new(unprocessed_transaction_storage: Option<&UnprocessedTransactionStorage>) -> Self {
+        if let Some(unprocessed_transaction_storage) = unprocessed_transaction_storage {
+            Self {
+                min_prioritization_fees_per_cu: unprocessed_transaction_storage
+                    .get_min_priority()
+                    .unwrap_or_default(),
+                max_prioritization_fees_per_cu: unprocessed_transaction_storage
+                    .get_max_priority()
+                    .unwrap_or_default(),
+            }
+        } else {
+            Self::default()
+        }
+    }
+
+    fn report(&self, id: u32, slot: Slot) {
+        datapoint_info!(
+            "banking_stage-leader_prioritization_fees_info_by_thread",
```

Comment:
if we add these, we should also have similar metrics for the min/max that the new scheduler sees.

Reply:
What should be done for that ?

Reply:
You're getting the min/max priority seen in the heap for legacy banking stage. Should do the same thing in the scheduler_controller, get the min/max priority in the heap at start of each schedule call. You'd need to add a fn for getting min/max on the `TransactionStateContainer`

Reply:
Updated :) thanks for indications